### DISCUSSION TITLE
What is the email address to contact ?

### DISCUSSION THREAD

**Discussion by shseok0724** [WalletConnect/walletconnect-monorepo#4046] [Developer Support] [answered]
I send you the email to 'contact@walletconnect.com'.
But I received 'contact@walletconnect.com' account does not exist.
```
550 5.1.1 The email account that you tried to reach does not exist. Please try double-checking the recipient's email address for typos or unnecessary spaces.
``` 
Do you have another email address?

Thank you.

**Comment by glitch-txs** [comment, marked as answer]
hi, you can send an email to support@walletconnect.com or devrel@walletconnect.com
